packages/react-conformance/package.json

```diff
@@ -38,10 +38,11 @@
     "react-dom": "16.8.6"
   },
   "dependencies": {
-    "tslib": "^1.10.0",
-    "react-is": "^16.6.3",
+    "doctrine": "^2.0.0",
```

Comment:
Why not the latest version? In any case, this package was deprecated two years ago https://github.com/eslint/doctrine

Reply:
The docblock conformance test was copied from react-northstar, so we used the same deps and versions as you have. We can use something else if you have another suggestion, but I'd prefer to fix that separately because this PR has been held up way too long.

Reply:
It's not a good reason to introduce deprecated packages to a new code. Please at least add a to replace this dependency.

Reply:
Despite the repo being archived, the [npm package](https://www.npmjs.com/package/doctrine) is not marked as deprecated and is extremely widely used (21 million downloads/week).

For alternatives, the next most widely used and similarly lightweight alternative that I could find is [`comment-parser`](https://www.npmjs.com/package/comment-parser) at 344k/week.